### PR TITLE
Improve descriptions

### LINKED ISSUE #162: [content] Jupiter description is weird
Jupiter's description is one word

https://climatescape.org/organizations/jupiter-intelligence

### PR DESCRIPTION
**Pull request by jasonLaster**
Fix #162 

- moves truncate to just the orgCard
- adds linkedin about

### CODE REVIEW

File: src/utils/airtable.js [hunk 1]
```diff
@@ -7,10 +7,8 @@ function getLogo(Logo, LinkedinProfile) {
   return logoSharp?.resize || logoSharp?.fuild || logoSharp?.fixed
 }
 
-function getDescription(description) {
-  const offset = 30
-  const index = description?.slice(offset).search(/\./) ?? -1
-  return index === -1 ? description : description.substr(0, index + offset + 1)
+function getAbout(About, LinkedinProfile) {
+  return LinkedinProfile?.[0]?.data.About || About || ""
```
Comment (bloudermilk) [inline src/utils/airtable.js]: "About" actually pulls from LinkedIn Profiles already using a formula on the Airtable site.

File: src/components/OrganizationCard.js [hunk 2]
```diff
@@ -19,6 +19,12 @@ function getLogoImage({ logo, photos, categories }) {
   return logo || photos[0] || cat?.cover || cat?.parent.cover
 }
 
+function truncateDescription(description) {
```
Comment (bloudermilk) [inline src/components/OrganizationCard.js]: This seems like the right place for this function

File: src/utils/airtable.js (hunk 1 above)
Comment (jasonLaster) [reply src/utils/airtable.js]: Nice, i guess i can simplify this :)

File: src/templates/organization.js [hunk 3]
```diff
@@ -183,6 +183,7 @@ export const query = graphql`
         }
         LinkedIn_Profiles {
           data {
+            About
```
Comment (bloudermilk) [inline src/templates/organization.js]: @jasonLaster if we could just remove this (it goes to the client) this is very ready to merge!

**Pull request merged**